client/ayon_zbrush/api/pipeline.py

```diff
@@ -193,8 +183,8 @@ def get_context_data(self):
 def containerise(
         name, context, namespace="", loader=None, containers=None):
     data = {
-        "schema": "openpype:container-2.0",
-        "id": AVALON_CONTAINER_ID,
+        "schema": "ayon:container-2.0",
```

Comment:
```suggestion
        "schema": "ayon:container-3.0",
```
I believe all container IDs used that are prefixed with ayon use 3.0